.github/workflows/bump-version.yml

```diff
@@ -23,20 +23,26 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: pnpm/action-setup@v2
```

Comment:
I would encourage to keep this PR smaller and move to pnpm in a separate PR

Reply:
I took it out. I had used it to make the examples work, but I now got them to work without. Plain npm does support workspaces, but not if the root of the repo is a public npm.